app/code/Magento/Ui/view/base/web/js/form/element/file-uploader.js

```diff
@@ -16,7 +16,8 @@ define([
     'Magento_Ui/js/form/element/abstract',
     'mage/backend/notification',
     'mage/translate',
-    'jquery/file-uploader'
+    'jquery/file-uploader',
+    'mage/adminhtml/tools'
```

Comment:
Why need tools?

Reply:
"Base64 is not defined" when try upload image -> caused by line 123 ->
```
 if (!file.id && file.name) {
                file.id = Base64.mageEncode(file.name);
            }
```

Reply:
This is core issue for ->  https://github.com/magento/magento2/issues/18688

Reply:
That's interesting, there are more js files using `Base64` without dependency on `mage/adminhtml/tools` - probably those files require to be fixed too